SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -171,11 +171,11 @@ void USpatialNetDriver::InitiateConnectionToSpatialOS(const FURL& URL)
 		Connection->LocatorConfig.PlayerIdentityToken = URL.GetOption(TEXT("playeridentity="), TEXT(""));
 		Connection->LocatorConfig.LoginToken = URL.GetOption(TEXT("login="), TEXT(""));
 		Connection->LocatorConfig.UseExternalIp = true;
-		Connection->LocatorConfig.WorkerType = GameInstance->GetSpatialWorkerType();
+		Connection->LocatorConfig.WorkerType = GameInstance->GetSpatialWorkerType().ToString();
 	}
 	else // Using Receptionist
 	{
-		Connection->ReceptionistConfig.WorkerType = GameInstance->GetSpatialWorkerType();
+		Connection->ReceptionistConfig.WorkerType = GameInstance->GetSpatialWorkerType().ToString();
```

Comment:
Maybe the `WorkerType` in the Receptionist and locator configs should be FNames as well? what do you guys think cc @mattyoung-improbable @Vatyx @tenevdev @alastairdglennie

Reply:
I don't see a reason why they should be FName, the one in the game instance is specifically because we expect it to be used in many equality checks.

Reply:
Maybe worth seeing just where `ReceptionistConfig.WorkerType` is used, could be similarly many equality checks

Reply:
I agree with Tencho that they don't need to be FNames.